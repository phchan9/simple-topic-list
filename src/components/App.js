import React from 'react';
import TopicListContainer from '../containers/TopicListContainer';
import AddTopicContainer from '../containers/AddTopicContainer';

const App = () => (
  <div clasName="flex-column">
    <AddTopicContainer />
    <TopicListContainer />
  </div>
);

export default App;
