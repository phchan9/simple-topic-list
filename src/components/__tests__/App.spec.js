import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';
import toJson from 'enzyme-to-json';

describe('App Component', () => {
  it('render correctly', () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
