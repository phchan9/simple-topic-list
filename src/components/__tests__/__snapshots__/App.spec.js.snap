// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component render correctly 1`] = `
<div
  className="flex-column"
>
  <Connect(AddTopic) />
  <Connect(TopicList) />
</div>
`;
